test/performance/scale_from_zero_test.go

```diff
@@ -160,44 +162,81 @@ func parallelScaleFromZero(t *testing.T, count int) ([]time.Duration, error) {
 	return durations, g.Wait()
 }
 
-func getStats(durations []time.Duration) *stats {
+func getRunStats(durations []time.Duration) *stats {
 	if len(durations) == 0 {
 		return nil
 	}
-	var avg time.Duration
-
-	for _, dur := range durations {
+	var avg, min, max time.Duration
+	min = durations[0]
+	max = durations[0]
+
+	for _, dur := range durations[1:] {
+		if dur < min {
+			min = dur
+		} else if dur > max {
+			max = dur
+		}
 		avg += dur
```

Comment:
why do you discard the first duration in the `avg` computation?

Reply:
fixed